java/net/URLEncoder/URLEncoderTest.cpp

```diff
@@ -28,14 +28,17 @@ extern "C" {
 #include "../../../kernel/test.h"
 }
 
-#include "URLEncoder.hpp"
+#include "../URLEncoder/URLEncoder.hpp"
 
 using namespace Java::Net;
 using namespace Java::Lang;
 
-TEST(JavaNet, UrlEncodeString) {
-//    unicode target = (unicode) "Quán ăn";
-//    String result = UrlEncoder::encode(target);
-//    unicode expect =  "Qu%c3%a1n+%c4%83n";
-//    ASSERT_STR(expect, result.toString());
-}
\ No newline at end of file
+TEST(JavaNet, URLEncodeString) {
```

Comment:
I think you should divide into 2 test cases
EncodeString and EncodeStringWithEncoding or something